equinix/provider.go

```diff
@@ -90,6 +91,8 @@ func Provider() *schema.Provider {
 			"equinix_fabric_networks":                  fabric_network.DataSourceSearch(),
 			"equinix_fabric_port":                      dataSourceFabricPort(),
 			"equinix_fabric_ports":                     dataSourceFabricGetPortsByName(),
+			"equinix_fabric_route_filter":              fabric_route_filter.DataSource(),
+			"equinix_fabirc_route_filters":             fabric_route_filter.DataSourceSearch(),
```

Comment:
There's a typo in the name of the search data source
```suggestion
			"equinix_fabric_route_filters":             fabric_route_filter.DataSourceSearch(),
```